### DISCUSSION TITLE
concurrent rendering name inconsistent

### DISCUSSION THREAD

**Discussion by JamesHutchison** [idom-team/idom#1202] [Problem]
All references to concurrent rendering use `ASYNC` and even the instructions in the changelog say to use the environment variable `REACTPY_ASYNC_RENDERING=true` but its actually `REACTPY_CONCURRENT_RENDERING=true`

```python
REACTPY_ASYNC_RENDERING = Option(
    "REACTPY_CONCURRENT_RENDERING",
    default=False,
    mutable=True,
    validator=boolean,
)
```

**Comment by Archmonger** [comment]
That's a bug. Do you want to PR to change it to ASYNC?

**Comment by JamesHutchison** [comment]
I'm going to hold off on messing with concurrent rendering as it doesn't benefit me. I don't do any I/O calls in the render logic under normal circumstances and all hooks are already async. The cpu portion of the rendering isn't actually in parallel because of how asyncio works.

**Comment by JamesHutchison** [comment]
I created a discussion around the concurrent rendering. Basically, I don't want to be the one to fix this because I feel like it's endorsing it when I'm not sure its providing value yet.

If it does provide value, then I suggest just having one code route. My presumption is that this is a config option since it may have bugs?
